apps/vault/package.json

```diff
@@ -34,21 +33,22 @@
   ],
   "license": "GPL-3.0-or-later",
   "description": "",
-  "devDependencies": {
-    "@aragon/apps-shared-migrations": "1.0.0",
-    "@aragon/cli": "^6.0.0",
-    "@aragon/test-helpers": "^2.1.0",
-    "@aragon/truffle-config-v4": "^1.0.1",
-    "eth-ens-namehash": "^2.0.8",
-    "eth-gas-reporter": "^0.2.0",
-    "ethereumjs-testrpc-sc": "^6.5.1-sc.0",
-    "ganache-cli": "^6.4.3",
-    "solidity-coverage": "0.6.2",
-    "solium": "^1.2.3",
-    "truffle": "4.1.14",
-    "truffle-extract": "^1.2.1"
-  },
   "dependencies": {
+    "@aragon/apps-vault": "^4.1.0",
     "@aragon/os": "4.2.0"
+  },
+  "devDependencies": {
+    "@aragon/buidler-aragon": "^0.2.0-beta.2",
+    "@aragon/contract-test-helpers": "0.0.1",
+    "@nomiclabs/buidler": "^1.0.2",
+    "@nomiclabs/buidler-etherscan": "^1.2.0",
+    "@nomiclabs/buidler-solhint": "^1.2.0",
+    "@nomiclabs/buidler-truffle5": "^1.1.2",
+    "@nomiclabs/buidler-web3": "^1.1.2",
+    "bignumber.js": "^9.0.0",
```

Comment:
Just wondering, since we're using `web3.toBN()` in the tests now (as we should be!), do we still need this package? Are we using it anywhere?

Reply:
This is a peer dependency of the Aragon Buidler plugin. If I recall correctly this was a types requirement. Regardless I'll make sure why we are not using web3 types instead.